client/src/components/Header/Header.tsx

```diff
@@ -54,15 +52,15 @@ const Header: FC<HeaderProps> = ({
             setModalVisible={setPresetModalVisible}
           />
         </Box>
-      );
-    }
-  };
+      )}
+    </>
+  );
 
   return suiteTitle ? (
     <AppBar color="default" className={styles.appbar}>
       <Toolbar className={styles.toolbar}>
         <Box display="flex" justifyContent="center">
-          <Link href="/">
+          <Link href="/inferno">
```

Comment:
This depends on where the app is loaded.  I'm not sure the best way to get that information here... but we do grab it in other parts of the app, like the API service:

https://github.com/inferno-framework/inferno-core/blob/ea465c850a9df7a29385c5e75e03ae4681694680/client/src/api/infernoApiService.ts#L1

Reply:
Gotcha! Will see what I can do